inotify.c

```diff
@@ -666,8 +666,7 @@ start_inotify()
 	if (setpriority(PRIO_PROCESS, 0, 19) == -1)
 		DPRINTF(E_WARN, L_INOTIFY,  "Failed to reduce inotify thread priority\n");
 	sqlite3_release_memory(1<<31);
-	av_register_all();
```

Comment:
Putting this issue here as reference. Removing this call is expected and does not affect behavior

https://github.com/leandromoreira/ffmpeg-libav-tutorial/issues/29